src/main/java/com/hackathonorganizer/hackathonwriteservice/hackathon/utils/HackathonMapper.java

```diff
@@ -0,0 +1,14 @@
+package com.hackathonorganizer.hackathonwriteservice.hackathon.utils;
+
+import com.hackathonorganizer.hackathonwriteservice.hackathon.model.Hackathon;
+import com.hackathonorganizer.hackathonwriteservice.hackathon.model.HackathonDto;
+import lombok.experimental.UtilityClass;
+
+@UtilityClass
+public class HackathonMapper {
+
+    public static HackathonDto mapHackathonToDto(Hackathon hackathon) {
```

Comment:
jeśli już mapper to wiemy że to jest HackatonMapper, więc mapować będzie hackaton, po co rozwlekać te nazwy, po porstu mapToDto